scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -238,24 +254,25 @@ Second, it would literally cause an error in R, which does not allow duplicate c
 One way to ensure that cell ids remain unique even after merging is to actually modify them by _prepending_ the relevant sample name. 
 For example, consider these barcodes for the `SCPCL000479` sample:
 
-```{r barcodes}
+```{r barcodes, live = TRUE}
 # Look at the column names for the `SCPCL000479` sample, for example
-colnames(sce_list$SCPCL000479) %>%
+colnames(sce_list$SCPCL000479) |>
```

Comment:
I'm 50/50 on this and the next chunk being live. You are just printing out an output and the scpca ids can be annoying and easy to mess up if you aren't used to the formatting.